packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -420,17 +437,24 @@ export default {
     onCaptchaVerified(response) {
       this.validationService.resetFormField("captcha", {
         reset: true,
-        value: response,
+        value: response
       });
     },
     onCaptchaExpired() {
       this.$refs.recaptcha.reset();
       this.validationService.resetFormField("captcha", {
         reset: true,
-        value: "",
+        value: ""
```

Comment:
Looks like a bit of a fight between IDEs/Prettier autoformatting. Was this autoformatted?

Reply:
yes it was autoformatted with prettier in my case